arch/arm/soc/st_stm32/stm32l0/gpio_registers.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2017 RnDity Sp. z o.o.
+ * Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _STM32L0X_GPIO_REGISTERS_H_
+#define _STM32L0X_GPIO_REGISTERS_H_
+
+/**
+ * @brief
+ *
+ * Based on reference manual:
+ *   STM32L0X advanced ARM ® -based 32-bit MCUs
+
+ *
+ * Chapter 8: General-purpose I/Os (GPIO)
+ * Chapter 9: System configuration controller (SYSCFG)
+ */
+
+struct stm32l0x_gpio {
+	u32_t moder;
+	u32_t otyper;
+	u32_t ospeedr;
+	u32_t pupdr;
+	u32_t idr;
+	u32_t odr;
+	u32_t bsrr;
+	u32_t lckr;
+	u32_t afr[2];
+	u32_t brr;
+};
+
+union syscfg_cfgr1 {
+	u32_t val;
+	struct {
+		u32_t mem_mode :2 __packed;
+		u32_t rsvd__2_7 :6 __packed;
+		u32_t adc_dma_rmp :1 __packed;
+		u32_t usart1_tx_dma_rmp :1 __packed;
+		u32_t usart1_rx_dma_rmp :1 __packed;
+		u32_t tim16_dma_rmp :1 __packed;
+		u32_t tim17_dma_rmp :1 __packed;
+		u32_t rsvd__13_15 :3 __packed;
+		u32_t i2c_pb6_fmp :1 __packed;
+		u32_t i2c_pb7_fmp :1 __packed;
+		u32_t i2c_pb8_fmp :1 __packed;
+		u32_t i2c_pb9_fmp :1 __packed;
+		u32_t i2c1_fmp :1 __packed;
+		u32_t rsvd__21 :1 __packed;
+		u32_t i2c_pa9_fmp :1 __packed;
+		u32_t i2c_pa10_fmp :1 __packed;
+		u32_t rsvd__24_25 :2 __packed;
+		u32_t usart3_dma_rmp :1 __packed;
+		u32_t rsvd__27_31 :5 __packed;
+	} bit;
+};
+
+union syscfg__exticr {
+	u32_t val;
+	struct {
+		u16_t exti;
+		u16_t rsvd__16_31;
+	} bit;
+};
+
+struct stm32l0x_syscfg {
```

Comment:
These registers are different in L0. I would replace that with https://pastebin.com/r3Vyed9b

Reply:
Will fix.

Reply:
This was fixed.